pkg/middlewares/headers/header.go

```diff
@@ -166,5 +167,16 @@ func (s *Header) isOriginAllowed(origin string) (bool, string) {
 		}
 	}
 
+	for _, item := range s.headers.AccessControlAllowOriginListRegex {
+		matched, err := regexp.MatchString(item, origin)
+		if err != nil {
+			log.WithoutContext().Debugf("an error occurred during origin parsing: %v", err)
```

Comment:
Maybe it's better to log an `Error` here, WDYT?

```suggestion
			log.WithoutContext().Errof("Unable to match origin: %v", err)
```